src/index.css

```diff
@@ -0,0 +1,68 @@
+:root {
```

Comment:
Make sure you remove all the unused code and styling from the template we installed. See "Clean Up Template" section in the assignment